packages/e2e-tests/specs/editor/various/block-directory-add.test.js

```diff
@@ -0,0 +1,166 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	createNewPost,
+	searchForBlock,
+	setUpResponseMocking,
+	getEditedPostContent,
+	getAllBlocks,
+} from '@wordpress/e2e-test-utils';
+
+/**
+ * Internal dependencies
+ */
+import { enableExperimentalFeatures } from '../../../experimental-features';
+
+// Urls to mock
+const SEARCH_URLS = [
+	'/__experimental/block-directory/search',
+	`rest_route=${ encodeURIComponent(
+		'/__experimental/block-directory/search'
+	) }`,
+];
+
+const INSTALL_URLS = [
+	'/__experimental/block-directory/install',
+	`rest_route=${ encodeURIComponent(
+		'/__experimental/block-directory/install'
+	) }`,
+];
+
+// Example Blocks
+const mockBlock1 = {
```

Comment:
I don't know if there are any style rules around this, but again referencing the embedding tests, the mock data was all-caps, which differentiated it from a regular variable — I'd suggest that here, too